### PR TITLE
feat: cli for syntax transformer

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Adding CLI command for syntax transformer

## References
Close https://github.com/openfga/syntax-transformer/issues/40

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: src/cmds/dslJson.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { Options as YargsOptions } from "yargs";
+
+import checkDSL from "../validator";
+import friendlySyntaxToApiSyntax from "../transformer";
+
+interface CommandArgs {
+  dslFile: string;
+}
+
+exports.command = "dslToJSON <dslFile>";
```
Comment (rhamzeh) [inline src/cmds/dslJson.ts:11]: This currently allows `@openfga/syntax-transformer dslToJSON <fileName>`

I think namespacing it by: `transform` allows for more flexibility in the future.

E.g. `@openfga/syntax-transformer transform --from=dsl <fileName>`

If the from is not passed, we try to get the from from the passed in file type

`to` is always the opposite

Later, we can add `--to` if we see a need to, but there is no need now.

Comment (rhamzeh) [reply src/cmds/dslJson.ts:11]: @matthewpereira any thoughts? This is just for transformation at the moment

Comment (adriantam) [reply src/cmds/dslJson.ts]: Updated with new syntax.

New syntax is 
```
npx @openfga/syntax-transformer transform --from=json --inputFile=test.json
npx @openfga/syntax-transformer transform --from=dsl --inputFile=test.dsl
```

Help menu shows
```
npx @openfga/syntax-transformer transform --help
syntax-transformer transform

Transform

Options:
  --version    Show version number                                     [boolean]
  --help       Show help                                               [boolean]
  --from       whether we want to transform from dsl or json
                                             [required] [choices: "dsl", "json"]
  --inputFile  Configuration file. It must be in DSL syntax. [string] [required]
```

File: src/index.ts [hunk 2]
```diff
@@ -31,3 +36,11 @@ export {
   apiSyntaxToFriendlySyntax,
   friendlySyntaxToApiSyntax,
 };
+
+yargs(hideBin(process.argv))
```
Comment (rhamzeh) [inline src/index.ts:40]: Just moving this to /bin/index.ts is left

Comment (adriantam) [reply src/index.ts]: Updated.  Confirmed by 
```adriantam@C02F5002MD6P syntax-transformer % npx @openfga/syntax-transformer transform
syntax-transformer transform

Transform

Options:
  --version    Show version number                                     [boolean]
  --help       Show help                                               [boolean]
  --from       whether we want to transform from dsl or json
                                             [required] [choices: "dsl", "json"]
  --inputFile  Configuration file. It must be in DSL syntax. [string] [required]

Missing required arguments: from, inputFile
```

**Review by rhamzeh — APPROVED** [review]

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**